tests/link/pytorch/test_basic.py

```diff
@@ -301,3 +302,25 @@ def test_pytorch_MakeVector():
     x_fg = FunctionGraph([], [x])
 
     compare_pytorch_and_py(x_fg, [])
+
+
+def test_pytorch_OpFromGraph():
+    x, y, z = matrices("xyz")
+    ofg_1 = OpFromGraph([x, y], [x + y])
+    OpFromGraph([x, y], [x * y, x - y])
+
+    # o1, o2 = ofg_2(y, z)
+    # out = ofg_1(x, o1) + o2
```

Comment:
I suppose this was testing nested OFG, we should add a test for that (can be a separate one)